Src/Localization/English/ButtonTutorial.html

```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
```

Comment:
New help pages should be added in separate commit as they are introducing new content.
It makes no sense to add completely new pages in commit that aims to make existing help files more uniform.

Reply:
I repeat. My main goal is to make it possible to easily localize all Open Shell messages (including group policy templates).
By the way. I made (a few years ago) for myself the ru-RU.dll library and it works correctly. I don't use the Immersive skin. ;-)
But there is what is called the shift of the motive to the goal - it became interesting for me to finish what I started and share it with everyone.
And so.
I absolutely do not care who will be indicated by the author of the changes in the source code. Make it as easy as possible for you. The main thing is the result! 
If any action is required from me, please let me know.
Once again, I would like to note that all the proposed changes/improvements in this PR are directly related to localization.
It's important.